packages/main/src/BusyIndicator.js

```diff
@@ -18,7 +18,7 @@ import busyIndicatorCss from "./generated/themes/BusyIndicator.css.js";
  * @public
  */
 const metadata = {
-	tag: "ui5-busyindicator",
+	tag: "ui5-busy-indicator",
```

Comment:
It's good to add "altTag: ui5-busyindicator" (you can see the usage of this metadata entry in DatePicker.js) not to break apps immediately but to allow more gradual transition.